test/Microsoft.OpenApi.Tests/Models/OpenApiSchemaTests.cs

```diff
@@ -643,6 +643,44 @@ public async Task WriteAsItemsPropertiesDoesNotWriteNull()
             """;
             Assert.True(JsonNode.DeepEquals(JsonNode.Parse(expected), JsonNode.Parse(actual)));
         }
+        [Fact]
+        public async Task SerializeConstAsEnumV30()
+        {
+            // Arrange
+            var schema = new OpenApiSchema
+            {
+                Type = JsonSchemaType.String,
+                Const = "foo"
+            };
+
+
+            // Act
+            var actual = await schema.SerializeAsJsonAsync(OpenApiSpecVersion.OpenApi3_0);
+
+            // Assert
+            var v3Node = JsonNode.Parse(actual);
```

Comment:
```suggestion
            var v3Node = Assert.NotNull(JsonNode.Parse(actual));
```
and you can remove the bang operator below